docs/public-api.asciidoc

```diff
@@ -355,31 +355,66 @@ preferably before other JS libraries:
 
 See the {apm-rum-ref}[JavaScript RUM agent documentation] for more information.
 
+[float]
+[[api-transaction-start-span-with-type]]
+==== `Span startSpan(String type, String subtype, String action)`
```

Comment:
Add subtype and action docs to `@CaptureSpan`

Reply:
ooooff, right 🤦‍♂️